apps/rush-lib/src/logic/TempProjectHelper.ts

```diff
@@ -0,0 +1,75 @@
+import { FileConstants, FileSystem, PosixModeBits } from '@rushstack/node-core-library';
+import * as tar from 'tar';
+import * as path from 'path';
+
+import { RushConfigurationProject } from '../api/RushConfigurationProject';
+import { RushConfiguration } from '../api/RushConfiguration';
+import { RushConstants } from './RushConstants';
+
+// The PosixModeBits are intended to be used with bitwise operations.
+/* eslint-disable no-bitwise */
+
+export class TempProjectHelper {
```

Comment:
This only applies to non-workspace repos, right?

Reply:
yes.